doc/overview.rst

```diff
@@ -240,7 +272,8 @@ The following are some examples to define estimators with Git support:
                                     train_instance_count=1,
                                     train_instance_type='ml.c4.xlarge')
 
-When Git support is enabled, users can still use local mode in the same way.
+Git support can be used not only for training jobs, but also for hosting models. The usage is the same as the above,
+and ``git_config`` should be provided when creating the ``FrameworkModel`` object.
```

Comment:
This seem like it needs more. There are more than one way to create a model, and I think a lot of customers won't even know that the model object they are creating inherits from FrameworkModel.